sql/geography.sql

```diff
@@ -0,0 +1,971 @@
+
+
+SET search_path = public, pg_catalog;
+DROP INDEX IF EXISTS public.wazimap_geography_name_36b79089_like;
+DROP INDEX IF EXISTS public.wazimap_geography_2fc6351a;
+DROP INDEX IF EXISTS public.wazimap_geography_84cdc76c;
+DROP INDEX IF EXISTS public.wazimap_geography_b068931c;
+ALTER TABLE IF EXISTS ONLY public.wazimap_geography DROP CONSTRAINT IF EXISTS wazimap_geography_pkey;
+ALTER TABLE IF EXISTS ONLY public.wazimap_geography DROP CONSTRAINT IF EXISTS wazimap_geography_geo_level_9a5128d2_uniq;
+ALTER TABLE IF EXISTS public.wazimap_geography ALTER COLUMN id DROP DEFAULT;
+DROP SEQUENCE IF EXISTS public.wazimap_geography_id_seq;
+DROP TABLE IF EXISTS public.wazimap_geography;
+
+SET search_path = public, pg_catalog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: wazimap_geography; Type: TABLE; Schema: public; Tablespace:
+--
+
+CREATE TABLE wazimap_geography (
+    id integer NOT NULL,
+    geo_level character varying(15) NOT NULL,
+    geo_code character varying(10) NOT NULL,
+    name character varying(100) NOT NULL,
+    year integer,
+    square_kms double precision,
+    parent_level character varying(15),
+    parent_code character varying(10),
+    long_name character varying(100)
+);
+
+
+--
+-- Name: wazimap_geography_id_seq; Type: SEQUENCE; Schema: public
+--
+
+CREATE SEQUENCE wazimap_geography_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+--
+-- Name: wazimap_geography_id_seq; Type: SEQUENCE OWNED BY; Schema: public
+--
+
+ALTER SEQUENCE wazimap_geography_id_seq OWNED BY wazimap_geography.id;
+
+ALTER TABLE ONLY wazimap_geography ALTER COLUMN id SET DEFAULT nextval('wazimap_geography_id_seq'::regclass);
+
+
+COPY wazimap_geography (name, geo_code, year, parent_level, long_name, geo_level, parent_code) FROM stdin WITH DELIMITER ',';
+Nepal,NP,2016,\N,Nepal,country,\N
+Province No. 1,1,2016,country,Province No. 1,province,NP
+Province No. 2,2,2016,country,Province No. 2,province,NP
+Province No. 3,3,2016,country,Province No. 3,province,NP
+Province No. 4,4,2016,country,Province No. 4,province,NP
+Province No. 5,5,2016,country,Province No. 5,province,NP
+Province No. 6,6,2016,country,Province No. 6,province,NP
+Province No. 7,7,2016,country,Province No. 7,province,NP
+Kathmandu,100,2016,province,Kathmandu,district,3
+Makwanpur,1006,2016,province,Makwanpur,district,3
+Parsa,1101,2016,province,Parsa,district,2
+Rautahat,1123,2016,province,Rautahat,district,2
+Bhojpur,1236,2016,province,Bhojpur,district,1
+Dhankuta,1299,2016,province,Dhankuta,district,1
+Morang,1326,2016,province,Morang,district,1
+Sankhuwasabha,1394,2016,province,Sankhuwasabha,district,1
+Sunsari,1467,2016,province,Sunsari,district,1
+Terathum,1475,2016,province,Terathum,district,
+Ilam,1538,2016,province,Ilam,district,1
+Jhapa,1581,2016,province,Jhapa,district,1
+Panchthar,1658,2016,province,Panchthar,district,1
+Taplejung,1658,2016,province,Taplejung,district,1
```

Comment:
1658 is a duplicate key here. There is a uniqueness constraint on `geo_code` and `geo_level`. I am not sure if there are other duplicates. When I tried to populate the table with this file, it errored here.

I used `cat geography.sql | psql -U wazimap_np -W wazimap_np` to try to populate the table using this file.

Reply:
(Accidentally hit "approve")